modules/mailchimp_ecommerce_commerce/mailchimp_ecommerce_commerce.module

```diff
@@ -490,6 +505,9 @@ function mailchimp_ecommerce_commerce_commerce_checkout_complete($order) {
   // Convert cart to order in MailChimp.
   mailchimp_ecommerce_delete_cart($order->order_id);
   mailchimp_ecommerce_add_order($order->order_id, $mc_order['customer'], $mc_order['order_data']);
+
+  // Delete the session variables.
+  unset($_SESSION['mc_cid'], $_SESSION['mc_landing_site']);
```

Comment:
Why are we deleting session variables here?

Reply:
This was per @taz77's suggestion that we clear these values, and it makes sense to do this during checkout completion. But I don't know if it's really necessary.

Reply:
My only concern with deleting these is what happens if the person buys something else immediately? Like, if you have an upsell on the checkout complete page and the person buys it?